packages/frames.js/src/core/middleware/framesjsMiddleware.ts

```diff
@@ -1,15 +1,21 @@
 import { FramesMiddleware, JsonValue } from "../types";
-import { isFrameRedirect, parseButtonInformationFromTargetURL } from "../utils";
+import {
+  isFrameRedirect,
+  parseButtonInformationFromTargetURL,
+  parseSearchParams,
+} from "../utils";
 
 type FramesjsMiddlewareContext = {
+  searchParams?: {
```

Comment:
this is already available on `ctx.currentURL.searchParams`

Reply:
yeah this has a less convenient URL api - `ctx.currentURL.searchParams.get('...')`. Would leave it as is

Reply:
unless you want to start formatting the `searchParams` in `currentURL` differently, which I suspect you wont